ui/partials/main/thActiveFiltersBar.html

```diff
@@ -8,10 +8,11 @@
       </span>
       <span ng-repeat="filter in filterBarFilters"
             class="filtersbar-filter">
-          <span title="Filter by {{ filter.field}}: {{ filter.value }}"
-                ng-if="filter.field == 'author'"><b>{{ filter.field }}:</b> {{filter.value.split('@')[0] | limitTo: 20}}</span>
-          <span title="Filter by {{ filter.field}}: {{ filter.value }}"
-                ng-if="filter !== 'author'"><b>{{ filter.field }}:</b> {{filter.value | limitTo: 12}}</span>
```

Comment:
I think this was the cause of the bug. `filter !==` should have been `filter.field !==` also.